client/src/main/scala/com.gu.contentapi.client/utils/QueryStringParams.scala

```diff
@@ -5,7 +5,8 @@ import java.net.URLEncoder
 object QueryStringParams {
   def apply(parameters: Iterable[(String, String)]) = {
     def encodeParameter(p: String): String = p match {
-      case other => URLEncoder.encode(other, "UTF-8")
+      //api-gateway IAM authorisation requires that spaces are encoded as %20, not +
+      case other => URLEncoder.encode(other, "UTF-8").replace("+", "%20")
```

Comment:
> UTF-8 is specified by https://tools.ietf.org/html/rfc3986#page-16
I like to annotate code with documentation of the source of requirements. I'm not sure if you follow a similar convention, so I leave it to you to decide whether you'd like to add it as a code comment.